src/main/ui/src/index.js

```diff
@@ -1,4 +1,11 @@
 
+import React from 'react';
```

Comment:
A bunch wrong from the merge conflict here.
<img width="1440" alt="Screen Shot 2022-09-01 at 10 02 40 PM" src="https://user-images.githubusercontent.com/7377390/188050495-359e3244-901a-45c0-be6a-1e60c6bce7ba.png">

Reply:
fixed